samples/samples-js/AddProductWithIdentityColumn/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const itemToInsert = {
+        "name": "productWithIdentity",
```

Comment:
You should be parsing the name and cost from the parameters like the C# function

Reply:
AM thinking of leaving the reading from body part so we support both get and post in the sample. What do you think?

Reply:
I would definitely agree that it would be great to do that at some point, but for now let's focus on getting the samples at parity and then we can come in and make targeted changes like that.

Reply:
all the samples are at parity with c# now, leaving that wouldn't do any harm and would avoid the rework.